src/app/item-page/bitstreams/request-a-copy/bitstream-request-a-copy-page.component.spec.ts

```diff
@@ -125,6 +143,8 @@ describe('BitstreamRequestACopyPageComponent', () => {
     beforeEach(() => {
       fixture = TestBed.createComponent(BitstreamRequestACopyPageComponent);
       component = fixture.componentInstance;
+      googleRecaptchaService.captchaVersion$ = captchaVersion$;
+      googleRecaptchaService.captchaMode$ = captchaMode$;
```

Comment:
It appears you've updated *every single test* in this file to now use Captchas.  We need to move this to a different block of tests to verify that the page works whether captchas are enabled or disabled.

Take a closer look at how this is done in `register-email-form.component.spec.ts`.  It moves ALL captcha tests into a single `describe` block instead of updating the `describe('init')` section near the top of the file.  See https://github.com/DSpace/dspace-angular/blob/main/src/app/register-email-form/register-email-form.component.spec.ts#L166-L193

Please make your updates in a similar way. We need this test file to prove that the page works whether captchas are enabled *or* disabled.  Right now you've updated it to only prove it works while they are enabled.